packages/authentication/src/__tests__/fixtures/strategies/jwt-strategy.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/authentication
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {inject} from '@loopback/context';
+import {HttpErrors, Request} from '@loopback/rest';
+import {AuthenticationStrategy, UserProfile} from '../../../types';
+import {JWTAuthenticationStrategyBindings} from '../keys';
+import {JWTService} from '../services/jwt-service';
+
+export class JWTAuthenticationStrategy implements AuthenticationStrategy {
+  name: string = 'jwt';
+
+  constructor(
+    @inject(JWTAuthenticationStrategyBindings.TOKEN_SERVICE)
+    public token_service: JWTService,
+  ) {}
+
+  async authenticate(request: Request): Promise<UserProfile | undefined> {
+    const token: string = this.extractCredentals(request);
+    const userProfile: UserProfile = await this.token_service.verifyToken(
+      token,
+    );
+    return userProfile;
+  }
+
+  extractCredentals(request: Request): string {
+    if (!request.headers.access_token) {
```

Comment:
From what I've read online, the typical way is checking the authorization header like `request.header.authorization`.

See "How do JSON Web Tokens work?" in https://jwt.io/introduction/

The `access_token` field is usually from the query, see our implementation in the [shopping cart example](https://github.com/strongloop/loopback4-example-shopping/blob/ee557f0dee0a9a5bd30e1f6b8f63b467d23162ca/packages/shopping/src/authentication-strategies/JWT.strategy.ts#L21).

Reply:
@jannyHou , I changed it to use the Authorization header with 'Bearer aaa.bbb.ccc' (where aaa.bbb.ccc is the JWT token ) instead.